mvc/models/article.model.js

```diff
@@ -24,11 +24,24 @@ exports.selectArticleComments = (articleId) => {
   return errorHandleDBQuery(query, [articleId], errorMsg).then((rows) => rows);
 };
 
+exports.insertCommentByArticleId = async (articleId, commentData) => {
+  if (!commentData.username || !commentData.body) {
```

Comment:
As the table constraints for comments body and author are ‘NOT NULL’ you should also get a psql error for this if no value is provided